packages/client/src/api/oauth.ts

```diff
@@ -8,7 +8,12 @@ export default class Oauth extends ApiBase implements OauthAPIType {
     const r = await this.POST('/api/v2/oauth/yandex', {
       body: JSON.stringify({ code, redirect_uri }),
     })
-    return await r.text()
+
+    if (r.ok) { 
+      return Promise.resolve({reason: 'ok'}) 
```

Comment:
А почему не написать просто вот так? Ведь `async` функция возвращает и так данные, обёрнутые в промис, ну тк я не вижу критичного тут, то просто комментом оставил
```suggestion
      return { reason: 'ok' } 
```